src/EnergyPlus/api/runtime.py

```diff
@@ -389,6 +391,14 @@ def callback_unitary_system_sizing(self, f: FunctionType) -> None:
         all_callbacks.append(cb_ptr)
         self.api.callbackUnitarySystemSizing(cb_ptr)
 
+    def clear_all_states(self) -> None:
```

Comment:
This is where this all started, somehow this function never made it to the Python bindings.